docs/guide/content_rendering.md

```diff
@@ -584,6 +584,41 @@ To avoid such situations, you can check if the Location is virtual using the `lo
     </div>
     ```
 
+    ### Landing Page template
+    Once published, Landing Page will be displayed using the template according to the `content_view` setting, see [View Matchers](#view-matchers). If you want to see the Landing Page displayed using a particular template in the edit mode of Landing Page Editor before publish, you need to configure additional settings as follow in `ezplatform.yml` configuration file.
```

Comment:
`Once publish, a Landing Page`...
`you need to configure the following additional settings in ezplatform...`